pupil-spa/src/app/accessibility-statement/accessibility-statement.component.html

```diff
@@ -74,39 +77,39 @@ <h3 class="heading-medium">Pupil site</h3>
         </tr>
         <tr>
           <td>Audio version</td>
-          <td>Not recommended for IOS 9.3 <br />
-            Not compatible with any version of IE and any version of Microsoft Edge v 14 and below
+          <td>Not recommended for iOS and Safari below version 12,
+            not compatible with any version of Internet Explorer and below version 14 of Microsoft Edge
           </td>
         </tr>
         <tr>
           <td>Remove on-screen number pad</td>
-          <td>Any browser/device (External keyboard required for tablets)</td>
+          <td>Any browser/device (external keyboard required for tablets)</td>
         </tr>
         </tbody>
       </table>
 
-      <h3 class="heading-medium">Teacher site</h3>
+      <h3 class="heading-medium">School administration website</h3>
 
-      <p>MTC does not currently have any built-in access arrangements for the teacher site.</p>
+      <p>The school administration website does not currently have any built-in access arrangements.</p>
 
-      <p>The MTC teacher site has been designed to be compatible with recent versions of the following screen readers:</p>
+      <p>The school administration website has been designed to be compatible with current versions (as of April 2023) of the following screen readers:</p>
 
       <ul class="list list-bullet">
         <li>job access with speech (JAWS)</li>
         <li>non-visual desktop access (NVDA)</li>
-        <li>voiceOver</li>
+        <li>voicevoer</li>
```

Comment:
Typo.
+
Apple call it VoiceOver not voiceover.  It's a proper noun and should be capitalised as it is a brand.

Reply:
good spot.  was intended to just lower the case, as per the spec.

this will need raising internally

Reply:
changed to VoiceOver